com.unity.render-pipelines.high-definition/Runtime/PostProcessing/Components/DepthOfField.cs

```diff
@@ -47,6 +49,22 @@ public enum DepthOfFieldResolution : int
         Full = 1
     }
 
+    /// <summary>
+    /// The source of the focus distance used in the Depth of Field Computations
```

Comment:
Options for the source of the focus distance HDRP uses in the depth of field calculations.